web/ui/dashboard/src/app/private/components/create-subscription/create-subscription.component.ts

```diff
@@ -273,10 +256,20 @@ export class CreateSubscriptionComponent implements OnInit {
 		this.showFilterDialog = true;
 	}
 
+	setupTransformDialog() {
```

Comment:
I don't understand this method, why aren't we calling the dialog close/open directly?